trustmanager/yubikey/yubikeystore.go

```diff
@@ -639,15 +639,15 @@ func (s *YubiKeyStore) ListKeys() map[string]string {
 }
 
 // AddKey puts a key inside the Yubikey, as well as writing it to the backup store
-func (s *YubiKeyStore) AddKey(keyID, role string, privKey data.PrivateKey) error {
-	added, err := s.addKey(keyID, role, privKey)
+func (s *YubiKeyStore) AddKey(keyInfo trustmanager.KeyInfo, privKey data.PrivateKey) error {
+	added, err := s.addKey(privKey.ID(), keyInfo.Role, privKey)
 	if err != nil {
 		return err
 	}
-	if added {
-		err = s.backupStore.AddKey(privKey.ID(), role, privKey)
+	if added && s.backupStore != nil {
```

Comment:
:+1: Awesome to not always have to write to a backup store!  This makes importing much nicer!